webapp/channels/src/components/claim/components/email_to_ldap.tsx

```diff
@@ -127,8 +128,8 @@ const EmailToLDAP = ({email, siteName, ldapLoginFieldName}: Props) => {
 
     const loginPlaceholder = ldapLoginFieldName || localizeMessage('claim.email_to_ldap.ldapId', 'AD/LDAP ID');
     const titleMessage = {id: t('claim.email_to_ldap.title'), defaultMessage: 'Switch Email/Password Account to AD/LDAP'};
-    const placeholderPasswordMessage = {id: t('claim.email_to_ldap.pwd'), defaultMessage: 'Password'};
-    const placeholderLdapMessage = {id: t('claim.email_to_ldap.ldapPwd'), defaultMessage: 'AD/LDAP Password'};
+    const placeholderPasswordMessage = intl.formatMessage({id: 'claim.email_to_ldap.pwd', defaultMessage: 'Password'});
```

Comment:
minor nit: Instead of defining the variable here, lets directly use it in the place
eg
```
<input
 placeholder={formatMessage(.......
 ```

Reply:
Applied